src/otx/recipe/detection/yolov9_c.yaml

```diff
@@ -45,6 +45,8 @@ overrides:
     train_subset:
       batch_size: 10
       transforms:
+        - class_path: otx.core.data.transform_libs.torchvision.PhotoMetricDistortion
+          enable: false
```

Comment:
This might replace YOLOXHSV with appropriate params. Let's put it close to each other, and replace later.